scripts/dowhop-profile-form-create.js

```diff
@@ -397,26 +373,28 @@ function socialMediaTW() {
   currentProfile = retrieveUrl(window.location.href) || firebase.auth().currentUser.uid;
   var profileRef = firebase.database().ref('app_users/' + currentProfile);
   profileRef.on('value', function(snap) {
-    console.log(currentProfile);
+    var prefix = "http://www.";
     let twitter = String(snap.val().profileSocialTW);
+    var link = prefix.concat(twitter)
     if (!snap.val().profileSocialTW) {
-      console.log("hello");
     } else {
-      window.open(twitter, '_blank');
+      console.log("twitter");
+      window.open(link, '_blank');
     }
   });
 }
 
 function socialMediaFB() {
   currentProfile = retrieveUrl(window.location.href) || firebase.auth().currentUser.uid;
   var profileRef = firebase.database().ref('app_users/' + currentProfile);
-  console.log(currentProfile);
   profileRef.on('value', function(snap) {
+    var prefix = "http://www.";
     let facebook = String(snap.val().profileSocialFB);
-    console.log(facebook);
+    var link = prefix.concat(facebook);
     if (!snap.val().profileSocialFB) {
     } else {
-      window.open(facebook, '_blank');
+      console.log("facebook");
```

Comment:
@tarekgouda @shar-ajmal Looking much better on mobile! Please comment out/remove any remaining `console.log`s before we merge into master.